python/opae.admin/opae/admin/fpga.py

```diff
@@ -572,6 +571,7 @@ def class_filter(cls, node):
 
 class fpga(fpga_base):
     _drivers = [fpga_region, fpga_base]
+    DEVICE_ROOT = 'class/fpga'
```

Comment:
should be DEVICE_ROOT = 'class/fpga_region'

Reply:
this is to support legacy driver, will remove when opae.admin is refactored.